conformance/tests/httproute-request-header-modifier.go

```diff
@@ -192,3 +194,28 @@ var HTTPRouteRequestHeaderModifier = suite.ConformanceTest{
 		}
 	},
 }
+
+var HTTPRouteInvalidRequestHeaderModifier = suite.ConformanceTest{
+	ShortName:   "HTTPRouteInvalidRequestHeaderModifier",
+	Description: "An HTTPRoute with invalid request header modifiers is not accepted",
+	Manifests:   []string{"tests/httproute-invalid-request-header-modifier.yaml"},
+	Test: func(t *testing.T, suite *suite.ConformanceTestSuite) {
+		ns := "gateway-conformance-infra"
+		gwNN := types.NamespacedName{Name: "same-namespace", Namespace: ns}
+
+		t.Run("HTTPRoutes that do intersect with listener hostnames", func(t *testing.T) {
```

Comment:
This probably was accidentally left over from another test

Reply:
Moot, removed test.